addons/point_of_sale/static/tests/tours/utils/chrome_util.js

```diff
@@ -9,6 +9,12 @@ export function clickMenuButton() {
         trigger: ".pos-rightheader button.fa-bars",
     };
 }
+export function clickLockButton() {
+    return {
+        content: "Click on the menu button",
+        trigger: ".pos-rightheader button.fa-lock",
+    };
+}
```

Comment:
you can't put this here. The button is only available when pos_hr is installed so it should be in pos_hr.